src/components/alert/alert.twig

```diff
@@ -1,7 +1,19 @@
 {% set tag_name = tag_name|default('div') %}
 
-<{{ tag_name }} class="c-alert{% if class %} {{ class }}{% endif %}">
+<{{ tag_name }} class="c-alert{% if full_width %} c-alert--full-width{% endif %}{% if floating %} c-alert--floating{% endif %}{% if class %} {{ class }}{% endif %}">
   <div class="c-alert__inner">
+    {% if block('icon')|trim or icon_name %}
+      <span class="c-alert--icon">
```

Comment:
This appears to be using modifier syntax (`.c-alert--icon`) instead of element syntax (`.c-alert__icon`).

Since this element is actually an icon _container_, we may want to call it `.c-alert__extra` for consistency with the Button component.